### PR TITLE
Sarthak work

### PR DESCRIPTION
**Pull request by sarthakKishorPatil**

### CODE REVIEW

File: allprogram/6buttontable project.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    <title>Document</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<style>
+    button{
+        margin: 0.5cm;
+        padding: 10px;
+        font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    }
+
```
Comment (HeenaPatil) [inline allprogram/6buttontable project.html]: Remove unnecessary spacing.

File: allprogram/6buttontable project.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    button{
+        margin: 0.5cm;
+        padding: 10px;
+        font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    }
+    
+</style>
+<body>
+   
+    <button name="1"  id="btn1"><b>1</b></button>
```
Comment (HeenaPatil) [inline allprogram/6buttontable project.html]: Id for paragraph tag is and button required ? Please remove unnecessary attribute .

File: allprogram/6buttontable project.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    <button name="6" id="btn6"><p id="output6"></p><b>6</b></button>
+    <button name="7" id="btn7"><p id="output7"></p><b>7</b></button>
+    <button name="8" id="btn8"><p id="output8"></p><b>8</b></button>
+    <button name="9" id="btn9"><p id="output9"></p><b>9</b></button>
+    <button name="10" id="btn10"><p id="output10"></p><b>10</b></button>
+    <p  id="output1"></p>
+    
+    <script>
+        
+        $(document).ready(function(){
```
Comment (HeenaPatil) [inline allprogram/6buttontable project.html]: function is not intended and remove unnecessary spacing . Add description of function.

File: allprogram/6buttontable project.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            $("button").click(function(){
+                var temp = parseInt( this.name);
+            $("#output1").append(function() {
+                  off="";
+
+                  for (let index = 1; index <=10 ; index++) 
+                  off=off+temp+"*"+index+"="+ index*temp +"<ol></ol>";
```
Comment (HeenaPatil) [inline allprogram/6buttontable project.html]: Variable names are not appropriate .

File: allprogram/6buttontable project.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+                  output1.innerHTML=off;
+        });
+        });
+        });
+
+                  
+
+        // $(document).ready(function(){
```
Comment (HeenaPatil) [inline allprogram/6buttontable project.html]: Remove unwanted code / commented code from source.

File: allprogram/Jbefore&Afteradd.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+ <img src="https://www.w3schools.com/images/w3jquery.gif" alt="jquery" width="100" height="140"  ><br><br>
+<!-- <button id="btn1">Insert before</button>
```
Comment (HeenaPatil) [inline allprogram/Jbefore&Afteradd.html]: Remove unwanted code .

File: allprogram/Jbefore&Afteradd.html [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+            $("img").before("<b>After</b>");
+        });
+    });
+
+    function afterText(){
+        var txt1="<b> I<b>";//create element with html
+            var txt2=$("<i></i>").text("love ");
+         var txt3=document.createElement("b");
+         txt3.innerHTML="jQuery!"
+            $("img").after(txt1,txt2,txt3);
```
Comment (HeenaPatil) [inline allprogram/Jbefore&Afteradd.html]: Variables names are not appropriate.

File: allprogram/Jbefore&Afteradd.html [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+ <img src="https://www.w3schools.com/images/w3jquery.gif" alt="jquery" width="100" height="140"  ><br><br>
+<!-- <button id="btn1">Insert before</button>
+<button id="btn2">Insert after</button> -->
+<p>click the button to insert text after the image.</p>
+<button onclick="afterText()">insert after</button>
+
+<script>
```
Comment (HeenaPatil) [inline allprogram/Jbefore&Afteradd.html]: Describe function functionality .

**Review by HeenaPatil — COMMENTED** [review]
Overall comment :
1. Files not correct.

**Comment by HeenaPatil** [comment]
**OVERALL  COMMENT**

**Comment by HeenaPatil** [comment]
1. File names are unappropiate.

**Comment by HeenaPatil** [comment]
2.Code is not cleaned

**Comment by HeenaPatil** [comment]
3. Variables names are appropriate .

**Comment by HeenaPatil** [comment]
4. please add proper commit comments.
5. Add readme.txt file in project .
6. review will not be done if commit is not from userid .

File: allprogram/6buttontable project.html (hunk 2 above)
Comment (sarthakKishorPatil) [reply allprogram/6buttontable project.html]: all id are required.

File: allprogram/6buttontable project.html (hunk 1 above)
Comment (sarthakKishorPatil) [reply allprogram/6buttontable project.html]: I had create new mathematicla table program

**Pull request merged**